packages/model-viewer/src/three-components/Damper.ts

```diff
@@ -14,27 +14,40 @@
  */
 
 export const SETTLING_TIME = 10000;  // plenty long enough
-const DECAY_MILLISECONDS = 50;
-const NATURAL_FREQUENCY = 1 / DECAY_MILLISECONDS;
-const NIL_SPEED = 0.0002 * NATURAL_FREQUENCY;
-
+const MIN_DECAY_MILLISECONDS = 0.001;
+export const DECAY_MILLISECONDS = 50;
 const $velocity = Symbol('velocity');
+const $naturalFrequency = Symbol('naturalFrequency');
 
 /**
  * The Damper class is a generic second-order critically damped system that does
  * one linear step of the desired length of time. The only parameter is
- * DECAY_MILLISECONDS, which should be adjustable: TODO(#580). This common
- * parameter makes all states converge at the same rate regardless of scale.
- * xNormalization is a number to provide the rough scale of x, such that
- * NIL_SPEED clamping also happens at roughly the same convergence for all
- * states.
+ * DECAY_MILLISECONDS. This common parameter makes all states converge at the
+ * same rate regardless of scale. xNormalization is a number to provide the
+ * rough scale of x, such that NIL_SPEED clamping also happens at roughly the
+ * same convergence for all states.
  */
 export class Damper {
   private[$velocity]: number = 0;
+  private[$naturalFrequency]: number = 0;
```

Comment:
You followed our style nicely here, but it turns out I just hadn't gotten around to fixing this yet. I've been removing symbols from all the classes that are not publicly visible, since they're pretty annoying to read (I changed this style rule when I took over the project, so there's still some left). Basically whenever I edit a file that still has them, I remove them (except for the mixins).

Reply:
lol, ok, done.